distribution/docker/build.gradle

```diff
@@ -11,39 +11,45 @@ testFixtures.useFixture()
 configurations {
   dockerPlugins
   dockerSource
+  ubiDockerSource
   ossDockerSource
+  ossUbiDockerSource
   restSpec
 }
 
 dependencies {
   dockerSource project(path: ":distribution:archives:linux-tar")
+  ubiDockerSource project(path: ":distribution:archives:linux-tar")
```

Comment:
Is this really necessary? Are we expecting a future scenario in which we'd actually use a different distribution archive for UBI vs non-UBI images? I think it'd be better just to ditch this complexity since a) we currently don't need it since we are using the same archive and b) we probably want to explicitly avoid such a scenario anyway because it's apt to cause us heartache.

Reply:
I pushed 21400bed2f23a5b0098a6d992402c094d422b29a.